site/docs/relations/physical_relations.md

```diff
@@ -67,10 +67,35 @@ The merge equijoin does a join by taking advantage of two sets that are sorted o
 | Left Input          | A relational input.                                                                                                                                                                                                     | Required                    |
 | Right Input         | A relational input.                                                                                                                                                                                                     | Required                    |
 | Left Keys           | References to the fields to join on in the left input.                                                                                                                                                                  | Required                    |
-| Right Keys          | References to the fields to join on in the right input.                                                                                                                                                            | Reauired                    |    
+| Right Keys          | References to the fields to join on in the right input.                                                                                                                                                                 | Required                    |
 | Post Join Predicate | An additional expression that can be used to reduce the output of the join operation post the equality condition. Minimizes the overhead of secondary join conditions that cannot be evaluated using the equijoin keys. | Optional, defaults true.    |
 | Join Type           | One of the join types defined in the Join operator.                                                                                                                                                                     | Required                    |
 
+
+
+## Single Join Operator
+
+The single match join works by enforcing that each row is involved in only a single match.  If this violated, an error
```

Comment:
Here you say "only a single match"  In the protobuf you say "at most one match".  Which is it?  At most one?  Or exactly one?

If it "at most one" then what happens if there is no match?  Is this an inner join (no row is emitted) or a left join (a row is emitted with the values from the left side and nulls on the right side)?